seed/tests/test_compliance_metrics_views.py

```diff
@@ -6,42 +6,35 @@
 """
 import json
 
-from django.test import TestCase
+from django.core import serializers
 from django.urls import reverse
 
-from seed.models import Column, ComplianceMetric, FilterGroup, User
-from seed.test_helpers.fake import FakeCycleFactory
-from seed.utils.organizations import create_organization
+from seed.models import ComplianceMetric, FilterGroup
+from seed.tests.util import AccessLevelTestCase
 
 
-class ComplianceMetricViewTests(TestCase):
+class ComplianceMetricViewTests(AccessLevelTestCase):
```

Comment:
Maybe the inherited call should be called `AccessLevelBaseTestCase`? To make it clear that it is inheriting from a base class...